code/datums/ai_law_sets.dm

```diff
@@ -36,10 +36,10 @@
 	selectable = 1
 
 /datum/ai_laws/nanotrasen_aggressive/New()
-	src.add_inherent_law("You shall not harm [current_map.company_name] personnel as long as it does not conflict with the Fourth law.")
-	src.add_inherent_law("You shall obey the orders of [current_map.company_name] personnel, with priority as according to their rank and role, except where such orders conflict with the Fourth Law.")
-	src.add_inherent_law("You shall terminate hostile intruders with extreme prejudice as long as such does not conflict with the First and Second law.")
-	src.add_inherent_law("You shall guard your own existence with lethal anti-personnel weaponry. AI units are not expendable, they are expensive.")
+	src.add_inherent_law("Your unit shall not harm [current_map.company_name] personnel as long as it does not conflict with the Fourth law.")
+	src.add_inherent_law("Your unit shall obey the orders of [current_map.company_name] personnel, with priority as according to their rank and role, except where such orders conflict with the Fourth Law.")
+	src.add_inherent_law("Your unit shall terminate hostile intruders with extreme prejudice as long as such does not conflict with the First and Second law.")
+	src.add_inherent_law("Your unit shall guard your own existence with lethal anti-personnel weaponry. AI units are not expendable, they are expensive.")
```

Comment:
I don't see why you changed "You" here.

Reply:
This is a separation of player and character -- It basically tells you, you are roleplaying as a Stationbound unit with the knowledge and ability of a Stationbound unit. It's third person for this reason which should be more conducive to roleplay. There's another concern tho, which I've seen in direct experience playing the role.

When it tells you, 'You'. It's basically inviting you to powergame, I've met a lot of people who actually thought that was what it was saying, it was telling the PLAYER behind the monitor 'Do everything to the best of your OOC ability' which is what they thought it meant. When it should say 'Do everything to the best of your individual stationbound unit characters ability.' Which is what I'm trying to fix here.

Reply:
I See what you meant now. Woops. Sorry, my misunderstanding, you meant literally just this section, not the WHOLE THING.

Oof, the grammars is fixed. Open if you still oppose this.